packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -190,3 +206,116 @@ export function getNetworkName(chainId: number) {
       return 'Unknown'
   }
 }
+
+export function getNetworkLogo(chainId: number) {
+  switch (chainId) {
+    // L1 networks
+    case ChainId.Mainnet:
+      return EthereumLogo
+
+    case ChainId.Rinkeby:
+      return EthereumLogo
+
+    case ChainId.Goerli:
+      return EthereumLogo
+
+    // L2 networks
+    case ChainId.ArbitrumOne:
+      return ArbitrumOneLogo
+
+    case ChainId.ArbitrumNova:
+      return ArbitrumNovaLogo
+
+    case ChainId.ArbitrumRinkeby:
+      return ArbitrumOneLogo
+
+    case ChainId.ArbitrumGoerli:
+      return ArbitrumOneLogo
+
+    default:
+      return EthereumLogo
+  }
+}
+
+type ExtendedWeb3Provider = Web3Provider & {
+  isMetaMask?: boolean
+  isImToken?: boolean
+}
+
+export const getSupportedNetworks = (chainId?: number) => {
+  // The list which will be available for network selection in navbar-dropdowns and unsupported-network-content
+  // If there is a chainId selected, it detects if it's a testnet and shows only those options
+
+  const mainNetworks = [
+    ChainId.ArbitrumOne,
+    ChainId.ArbitrumNova,
+    ChainId.Mainnet
+  ]
+  const testNetworks = [ChainId.ArbitrumGoerli, ChainId.Goerli]
+  const isTestnet = chainId ? isNetwork(chainId).isTestnet : false
```

Comment:
can we do a check for undefined value in `isNetwork` instead?

on another note, are we going to detect Sepolia too and recommend them to Goerli if they are on Sepolia?

Reply:
That's a good point, we can put Sepolia in there, too